src/components/v5/shared/Navigation/ColonySwitcher/partials/JoinedColonyItem/JoinedColonyItem.tsx

```diff
@@ -14,11 +15,12 @@ const JoinedColonyItem = ({
   name,
   colonyAddress,
   tokenSymbol,
+  isActiveColony,
 }: JoinedColonyItemProps) => {
   return (
     <button
       type="button"
-      className="group flex h-[37px] flex-row items-center justify-start gap-3 rounded-lg p-2 hover:bg-gray-50"
+      className="group flex h-[50px] w-full flex-row items-center justify-start gap-3 rounded-lg px-4 py-8 hover:bg-gray-50 md:px-2 md:py-0"
```

Comment:
Pretty minor, but since testing this PR is all about eagle design eyes, this className should have `rounded` instead of `rounded-lg` I believe!

Reply:
You're totally right! I shall update 👌